content/config/containers/logging/fluentd.md

```diff
@@ -150,24 +150,24 @@ aggregate store.
 
 1.  Write a configuration file (`test.conf`) to dump input logs:
 
-    ```none
+    ```text
     <source>
       @type forward
     </source>
 
     <match *>
       @type stdout
     </match>
-    ```none
+    ```
 
 2.  Launch Fluentd container with this configuration file:
 
-    ```none
+    ```text
```

Comment:
Looks like these should've been `console`?

Reply:
yeah noticed too late 🙈